### ISSUE TITLE
C-x key sequence in Emacs mapping never ends

### ISSUE THREAD

**Issue by daigot** [k0kubun/xremap#149]
Hello,

Thank you for your great work.
I'm trying xremap with emacs.yml and I encountered a curious behavior in key sequence.
It looks C-x key sequence never ends.

## Environment
- Ubuntu 22.04 (XOrg session)
- xremap-linux-x86_64-x11 0.7.3

## Steps to reproduce
- ```xremap emacs.yml```
- run "Text Editor" (gedit)
- press C-x C-s (save) and cancel it
- pressing C-f opens "Open Files" dialog, pressing h causes selecting all

Thanks.

**Comment by k0kubun** [comment]
> xremap emacs.yml

Is that exactly [example/emacs.yml](https://github.com/k0kubun/xremap/blob/bc2c7aaa10a7a0a81d92c6f57e3424ed0e324e1c/example/emacs.yml)?

**Comment by daigot** [comment]
Thank you for quick replay.

> At least I don't understand what you mean by "cancel it".

I'm sorry for my ambiguous explanation.
This means that close the save dialog (= click cancel button).

> Is that exactly [example/emacs.yml](https://github.com/k0kubun/xremap/blob/bc2c7aaa10a7a0a81d92c6f57e3424ed0e324e1c/example/emacs.yml)?

Yes.
It is exactly that file.

Debug output is following.
I pressed C-x C-s (and close the save dialog) and then pressed C-f
```
$ RUST_LOG=debug xremap emacs.yml 
Selecting devices from the following list:
------------------------------------------------------------------------------
/dev/input/event0 : Power Button
/dev/input/event1 : Power Button
/dev/input/event10: HD-Audio Generic HDMI/DP,pcm=8
/dev/input/event11: HD-Audio Generic HDMI/DP,pcm=9
/dev/input/event12: HD-Audio Generic Front Mic
/dev/input/event13: HD-Audio Generic Rear Mic
/dev/input/event14: HD-Audio Generic Line
/dev/input/event15: HD-Audio Generic Line Out
/dev/input/event16: HD-Audio Generic Front Headphone
/dev/input/event2 : Video Bus
/dev/input/event3 : ASRock LED Controller
/dev/input/event4 : Chicony USB Keyboard
/dev/input/event5 : Chicony USB Keyboard System Control
/dev/input/event6 : Chicony USB Keyboard Consumer Control
/dev/input/event7 : Logitech M705
/dev/input/event8 : HD-Audio Generic HDMI/DP,pcm=3
/dev/input/event9 : HD-Audio Generic HDMI/DP,pcm=7
------------------------------------------------------------------------------
Selected keyboards automatically since --device options weren't specified:
------------------------------------------------------------------------------
/dev/input/event4 : Chicony USB Keyboard
------------------------------------------------------------------------------
[2022-08-22T09:48:07Z DEBUG xremap::event_handler] => 0: KEY_ENTER
[2022-08-22T09:48:07Z DEBUG xremap::event_handler] 0: KEY_ENTER
[2022-08-22T09:48:20Z DEBUG xremap::event_handler] => 1: KEY_RIGHTCTRL
[2022-08-22T09:48:20Z DEBUG xremap::event_handler] 1: KEY_RIGHTCTRL
[2022-08-22T09:48:21Z DEBUG xremap::event_handler] => 2: KEY_RIGHTCTRL
[2022-08-22T09:48:21Z DEBUG xremap::event_handler] 2: KEY_RIGHTCTRL
[2022-08-22T09:48:21Z DEBUG xremap::event_handler] => 2: KEY_RIGHTCTRL
[2022-08-22T09:48:21Z DEBUG xremap::event_handler] 2: KEY_RIGHTCTRL
[2022-08-22T09:48:21Z DEBUG xremap::event_handler] => 1: KEY_X
[2022-08-22T09:48:21Z DEBUG xremap::event_handler] => 0: KEY_X
[2022-08-22T09:48:21Z DEBUG xremap::event_handler] 0: KEY_X
[2022-08-22T09:48:21Z DEBUG xremap::event_handler] => 1: KEY_S
[2022-08-22T09:48:21Z DEBUG xremap::event_handler] 1: KEY_S
[2022-08-22T09:48:21Z DEBUG xremap::event_handler] 0: KEY_S
[2022-08-22T09:48:21Z DEBUG xremap::event_handler] => 0: KEY_S
[2022-08-22T09:48:21Z DEBUG xremap::event_handler] 0: KEY_S
[2022-08-22T09:48:21Z DEBUG xremap::event_handler] => 0: KEY_RIGHTCTRL
[2022-08-22T09:48:21Z DEBUG xremap::event_handler] 0: KEY_RIGHTCTRL
[2022-08-22T09:48:27Z DEBUG xremap::event_handler] => 1: KEY_RIGHTCTRL
[2022-08-22T09:48:27Z DEBUG xremap::event_handler] 1: KEY_RIGHTCTRL
[2022-08-22T09:48:27Z DEBUG xremap::event_handler] => 1: KEY_F
[2022-08-22T09:48:27Z DEBUG xremap::event_handler] 1: KEY_O
[2022-08-22T09:48:27Z DEBUG xremap::event_handler] 0: KEY_O
[2022-08-22T09:48:28Z DEBUG xremap::event_handler] => 0: KEY_F
[2022-08-22T09:48:28Z DEBUG xremap::event_handler] 0: KEY_F
[2022-08-22T09:48:28Z DEBUG xremap::event_handler] => 0: KEY_RIGHTCTRL
[2022-08-22T09:48:28Z DEBUG xremap::event_handler] 0: KEY_RIGHTCTRL
```

**Comment by k0kubun** [comment]
Thank you for providing the log. Fixed this in v0.7.4 :man_facepalming:

**Comment by daigot** [comment]
Works as expected.
Thank you for fixing the issue so quickly. :pray:
